src/nativestore/NodeBlock.cpp

```diff
@@ -302,11 +299,11 @@ std::list<NodeBlock> NodeBlock::getEdges() {
             node = NodeBlock::get(currentRelation->source.address);
             currentRelation = currentRelation->nextDestination();
         } else {
-            throw "Error: Unrecognized relation for " + std::to_string(this->addr) + " in relation block " +
-                std::to_string(currentRelation->addr);
+            throw std::logic_error("Error: Unrecognized relation for " + std::to_string(this->addr) +
```

Comment:
Rather than throwing an error, lets log it. Throwing may make this error getting unnoticed.

Reply:
Changed to log error

Reply:
Ok